src/grafana_agent.py

```diff
@@ -368,7 +368,7 @@ def update_dashboards(
         try:
             dashboards = dashboards_func
         except NotImplementedError:
-            logger.debug("Dashboard forwarding is not yet enabled for k8s grafana-agent")
+            logger.error("Dashboard forwarding is not yet enabled for this substrate!")
```

Comment:
This feels unnecessary. Let's just get rid of it.

Reply:
ack. removed